README.md

```diff
@@ -1,25 +1,57 @@
+
+Code and content for [nixos.org](https://nixos.org) website.
+
+
+# Help us!
+
+There are many ways how you can help:
+
+- if you are familiar with CSS look at the [issues tagged with `design` tag](https://github.com/NixOS/nixos-homepage/issues?q=is%3Aissue+is%3Aopen+label%3Adesign).
+- if you are an native English speaker or just a person that is very good with words, please look at the [issues tagged with `content` tag](https://github.com/NixOS/nixos-homepage/issues?q=is%3Aissue+is%3Aopen+label%3Acontent)
+- if you are developer and just eager to fix stuff please look at the [issues tagged with `bug` tag](https://github.com/NixOS/nixos-homepage/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+
+If you feel lost where and how to contribute, ask [marketing team](https://nixos.org/teams/marketing.html) on [`#nixos-marketing` channel on IRC](irc://irc.freenode.net/#nixos-marketing).
+
+
+# How to help?
+
+For quick fixes you can use gitpod.io: [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/nixos/nixos-homepage)
+
+To run local development instance follow this steps:
 
     $ git clone git@github.com:NixOS/nixos-homepage.git
     $ cd nixos-homepage
     $ nix-shell
-    [nix-shell]$ make
-    [nix-shell]$ python -m http.server
 
-then open http://127.0.0.1:8000/index.html
+      To start developing run:
+          python run.py
+
+      and open browser on:
+          https://127.0.0.1:8000
+
+      It will rebuild the website on each change.
+
+    [nix-shell]$ python run.py
+
+Open your browser at: http://127.0.0.1:8000/index.html
+
+In order for browser to automatically refresh install [Livereload extension](http://livereload.com/extensions/) for you browser.
+
+Before creating a pull request make sure that `nix-build` runs successfully.
+
+
+## Binary cache
 
-To automatically rebuild on every change:
+It can take some time to enter the development environment. To speed up and avoid building from source, you can use a binary cache that is also use to speed up our GitHub Actions.
 
-    [nix-shell]$ fd | entr make
+Add the following to the `/etc/nix/nix.conf` or `~/.config/nix/nix.conf`:
```

Comment:
Hm, editing `/etc/nix/nix.conf` on NixOS is not really going to work. How can we rephrase this in a way that handles NixOS, and Non-NixOS alike?

Quite out of scope, but it would be best if there was a centralized page that deals with how to edit the Nix configuration, so we could link to it, with which configuration option to edit.

Or maybe even a documentation section about adding *binary caches* outright?